src/__tests__/core/client/index.test.js

```diff
@@ -0,0 +1,50 @@
+import Immutable from 'immutable';
+import { initClient } from '../../../core/client';
+
+describe('core/client/index', () => {
+  describe('initClient', () => {
+    it('loads password policy correctly without a password_complexity_options option', () => {
+      const client = {
+        strategies: [
+          {
+            name: 'auth0',
+            connections: [
+              {
+                name: 'Username-Password-Authentication',
+                passwordPolicy: 'low'
+              }
+            ]
+          }
+        ]
+      };
+      const result = initClient(Immutable.fromJS({}), client).toJS();
+      expect(result.client.connections.database[0].passwordPolicy).toMatchObject({
+        length: {
+          minLength: 6
```

Comment:
Please add the same tests for the remaining policies.

Reply:
That would be just a test for an external resource, not sure that's needed.

Reply:
How can I trust that your code will behave the same when the policy used is other than `low`? Please add them.

Reply:
pushed